### PR TITLE
Added write!() to Image, upgrade imageMagick to ImageMagick-6.9.2-8-portable-Q16-x86

### PR DESCRIPTION
**Pull request by ryancastro**
I realized for my my usage of win32screenshot, there were many occasions where overwriting my existing files made sense. I added a `write!()` method to support that.

### CODE REVIEW

File: lib/win32/screenshot/image.rb [hunk 1]
```diff
@@ -27,6 +27,17 @@ def initialize(blob, width, height)
...
+      end
+      
+      #Writes image to disk, writing over existing copy if it exists.
+      def write!(file_path)
+        write_file file_path
+      end
+      
+      private
+      
+      def write_file(file_path)
```
Comment (jarmo) [inline lib/win32/screenshot/image.rb]: No need to have this separate method here. Just inline it to `#write!` and call that method from `#write`. Makes perfect sense to me.

**Comment by ryancastro** [comment]
I have updated the commit with your feedback. Honestly, I looked at your style, design, and feedback to other pull requests and rolled the dice as to which method you'd like the most, and the one that would make _you_ the happiest. I chose poorly. :) Regardless, I have made the suggested change and agree with you. How does that look?

**Comment by jarmo** [comment]
Tried to merge changes in, but I'm seeing failing spec:
```
Failures:

  1) Win32::Screenshot::Image saving the image to the disk allowing to overwrite existing files when desired #write!
     Failure/Error: File.size(file_path).should != content.size
     RuntimeError:
       RSpec does not support `should != expected`.  Use `should_not == expected` instead.
     # ./spec/win32/screenshot/image_spec.rb:68:in `block (4 levels) in <top (required)>'
```

Is this spec passing for you? Did you run specs with `bundle exec rake spec` command instead of using a global (possibly different) version of `rspec` command?

**Comment by jarmo** [comment]
I had to fix that specific spec, change some other specs as well (using notepad instead of a calculator, which is not working really well under Windows 10), upgrade bundled ImageMagick to even newer version because I did have some warnings otherwise/images not captured correctly (spec/tmp had only black images). I've also updated mini_magick dependency. Anyway, thanks for your initial work, which has now been merged and released. Let me know if you're still seeing any problems.

**Pull request merged**